install/create-kafka-topics.sh

```diff
@@ -1,12 +1,24 @@
 echo "${_group}Creating additional Kafka topics ..."
 
-# NOTE: This step relies on `kafka` being available from the previous `snuba-api bootstrap` step
+$dc up --no-build --no-recreate kafka
+
+while [ true ]
+do
+    $kafka_healthy=$(docker compose ps kafka | grep 'healthy')
```

Comment:
Ah sorry, this should be `$dc ps kafka`

```suggestion
    $kafka_healthy=$($dc ps kafka | grep 'healthy')
```